packages/core/src/checkbox/checkbox.scss

```diff
@@ -20,27 +22,20 @@
   gap: var(--_gap);
   line-height: var(--_line-height);
   outline: none;
+  padding: var(--_padding);
 }
 
-:host(:disabled) {
+:host([disabled]) {
```

Comment:
I thought with `ElementInternals` we could use `:disabled`? 

From `button.scss`:

```sass
:host(:where(:disabled, [internals-disabled])) {
  --_cursor: default;
  pointer-events: none;
}
```

Reply:
`[internals-disabled]` is for the element internals polyfill.

Reply:
:disabled doesn't work in safari

Reply:
That's what the `[internals-disabled]` is for :)

Reply:
right, that does work... :D 
I only tried it with :disabled i think